sdk/attestation/azure-security-attestation/azure/security/attestation/_models.py

```diff
@@ -5,59 +5,69 @@
 # --------------------------------------------------------------------------
 
 import base64
-import json
-from cryptography.hazmat.primitives import serialization
+from typing import Any, Dict, List, Type, TypeVar, Union
+from json import JSONDecoder, JSONEncoder
+from datetime import datetime
 
 from cryptography.hazmat.primitives.asymmetric import padding
 from cryptography.hazmat.primitives.hashes import SHA256
-from ._common import Base64Url
+from cryptography.exceptions import InvalidSignature
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurvePrivateKey
+from cryptography.hazmat.primitives.serialization import Encoding
+from cryptography.x509 import Certificate, load_pem_x509_certificate
+from cryptography.hazmat.primitives.asymmetric.rsa import RSAPrivateKey, RSAPublicKey
+from six import string_types, raise_from
+
+from ._common import (
+    base64url_decode,
+    base64url_encode,
+    pem_from_base64,
+    validate_signing_keys,
+)
 from ._generated.models import (
-    PolicyResult as GeneratedPolicyResult, 
+    PolicyResult as GeneratedPolicyResult,
     AttestationResult as GeneratedAttestationResult,
     StoredAttestationPolicy as GeneratedStoredAttestationPolicy,
+    PolicyCertificatesModificationResult as GeneratedPolicyCertificatesModificationResult,
     JSONWebKey,
     CertificateModification,
-    AttestationType,
-    PolicyModification
+    PolicyModification,
 )
-from typing import Any, Callable, Dict, List, Type, TypeVar, Generic, Union
-from cryptography.hazmat.backends import default_backend
-from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurvePrivateKey
-from cryptography.hazmat.primitives.serialization import Encoding, PublicFormat
-from cryptography.x509 import Certificate, load_der_x509_certificate
-from cryptography.hazmat.primitives.asymmetric.rsa import RSAPrivateKey, RSAPublicKey
-from json import JSONDecoder, JSONEncoder
-from datetime import datetime
 
```

Comment:
These imports can be removed from the `from typing import ...` line above, since they're only needed when type checking (this is usually the case for `typing` types, so it may apply to other files)
```suggestion

if TYPE_CHECKING:
    # pylint: disable=ungrouped-imports
    from typing import Any, Dict, List, Type, Union

```